api/lib/infrastructure/exports/campaigns/campaign-profile-collection-result-line.js

```diff
@@ -6,12 +6,13 @@ const NOT_SHARED = 'NA';
 
```

Comment:
on devrait aussi traduire "NA" non ?

Reply:
No Attribution ? 😄

Reply:
ha mais c'est déjà de l'anglais en fait ^^ ça veut dire "not available" ^^, par contre le mettre dans les fichiers de trad ce serait mieux je pense